rmsprop.lua

```diff
@@ -25,27 +22,24 @@ function optim.rmsprop(opfunc, x, config, state)
     -- (0) get/update state
     local config = config or {}
     local state = state or config
-    local lr = config.learningRate or 1e-4
-    local alpha = config.alpha or 0.998
+    local lr = config.learningRate or 1e-2
+    local alpha = config.alpha or 0.95
     local epsilon = config.epsilon or 1e-8
-    local epsilon2 = config.epsilon2 or 1e-8
-    local max_gain = config.max_gain or 1000
-    local min_gain = config.min_gain or 1e-8
 
     -- (1) evaluate f(x) and df/dx
     local fx, dfdx = opfunc(x)
 
     -- (2) initialize mean square values and square gradient storage
-    state.m = state.m or torch.Tensor():typeAs(dfdx):resizeAs(dfdx):fill(epsilon)
+    state.m = state.m or torch.Tensor():typeAs(dfdx):resizeAs(dfdx):zero()
     state.tmp = state.tmp or x.new(dfdx:size()):zero()
 
-    -- (3) calculate new mean squared values
+    -- (3) calculate new (leaky) mean squared values
     state.m:mul(alpha)
-    state.m:addcmul(1.0-alpha,dfdx,dfdx):add(epsilon2)
+    state.m:addcmul(1.0-alpha,dfdx,dfdx)
 
     -- (4) perform update
-    state.tmp:copy(state.m):pow(-0.5):clamp(min_gain, max_gain)
-    x:add(-lr, state.tmp)
+    state.tmp:copy(state.m):sqrt()
```

Comment:
Thanks, I saw the `:copy()` pattern in all the other scripts (e.g. Adagrad, Adam, etc.) so I assumed it was an intentional. Incorporated.